### PR TITLE
Move examples to their own repo

### PR DESCRIPTION
**Pull request by finn-tbd**
## What type of PR is this? (check all applicable)

- [x] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR moves the examples code into the [tbd-examples](https://github.com/TBD54566975/tbd-examples) repo.

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by EbonyLouis** [comment]
They're separated by languages should be separate them by project too? like web5 and tbDEX?

**Comment by angiejones** [comment]
All tbDEX examples use web5 by default. I'd prefer if the README files only listed tbDEX and not the comma-delimited `tbDEX, web5`.

File: site/docs/docs-index.js
Comment (angiejones) [inline site/docs/docs-index.js:1]: These 2 links also need to be updated

image

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/docs-index.js
Comment (finn-tbd) [reply site/docs/docs-index.js:1]: ooo good catch! i updated the iOS wallet link, but we haven't moved the pfi-exemplar yet ([ongoing discussion on that](https://github.com/TBD54566975/tbdex-pfi-exemplar/pull/52)) so I'm not going to update that link yet. I left a note on that side to remind us to update this one when we decide what to do over there.

**Review by angiejones — APPROVED** [review]

**Pull request merged**